Cargo.toml

```diff
@@ -49,6 +50,9 @@ serde_derive = "1.0.0"
 serde = "1.0.0"
 clippy = { version = "=0.0.142", optional = true }
 chrono = "0.4.0"
+pulldown-cmark = { version = "0.0.15", default-features = false }
+ammonia = "0.5.0"
+syntect = "1.7.1"
```

Comment:
I'm not sure I understand... The current [lock file](https://github.com/rust-lang/crates.io/blob/master/Cargo.lock#L408) references `diesel 0.14.1` despite having `diesel = 0.14.0` in the [Cargo.toml](https://github.com/rust-lang/crates.io/blob/master/Cargo.toml#L43). Where is the behavior you describe documented ?